components/style/values/generics/counters.rs

```diff
@@ -8,7 +8,6 @@
 use crate::computed_values::list_style_type::T as ListStyleType;
 #[cfg(feature = "gecko")]
 use crate::values::generics::CounterStyle;
-#[cfg(feature = "gecko")]
```

Comment:
This and other `cfg`s changed here should probably be something like `#[cfg(any(feature = "gecko", feature = "servo-layout-2020"))]`, since layout 2013 still doesn’t support this.